### PR TITLE
Add support for Infomaniak as provider

### PR DESCRIPTION
**Pull request by diizzyy**
Add support for Infomaniak(.com), follows the same schema as Google Domains

### CODE REVIEW

**Comment by diizzyy** [comment]
Not sure what the CI is not happy about

File: internal/provider/factory.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/TimothyYe/godns/internal/provider/hetzner"
```
Comment (TimothyYe) [inline internal/provider/factory.go:15]: Would u pls help to format this file?

Comment (diizzyy) [reply internal/provider/factory.go:15]: Sure but I can't find any issues

Comment (TimothyYe) [reply internal/provider/factory.go:15]: ok, let me merge it first and check it.

**Comment by TimothyYe** [comment]
Thanks for your contribution.

**Comment by TimothyYe** [comment]
Here is the reason, the go lint CI checks the order of the imported packages:

image

**Comment by diizzyy** [comment]
Ahh, cool! Thanks! :)

**Pull request merged**